pkg/storage/buckets/store.go

```diff
@@ -29,7 +33,6 @@ type Store interface {
 	Put(ctx context.Context, bucket string, inMeta Meta) (meta Meta, err error)
 	Delete(ctx context.Context, bucket string) (err error)
 	List(ctx context.Context, startAfter, endBefore string, limit int) (items []ListItem, more bool, err error)
-	GetObjectStore(ctx context.Context, bucketName string) (store objects.Store, err error)
```

Comment:
im curious why remove `GetObjectStore`?

Reply:
We no longer have an ObjectStore on the BucketStore struct

Reply:
but why? why remove object store?